cypress/integration/plugins/custom-import-map-dashboards/add_saved_object.spec.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { BASE_PATH } from '../../../utils/constants';
+
+describe('Add flights dataset saved object', () => {
+  before(() => {
+    cy.visit(`${BASE_PATH}/app/maps-dashboards`, {
+      retryOnStatusCodeFailure: true,
+      timeout: 60000,
+    });
+    cy.get('div[data-test-subj="indexPatternEmptyState"]', { timeout: 60000 })
+      .contains(/Add sample data/)
+      .click();
+    cy.get('div[data-test-subj="sampleDataSetCardflights"]', { timeout: 60000 })
+      .contains(/Add data/)
+      .click();
+    cy.wait(60000);
```

Comment:
This should be avoided. This is an arbitrary wait time that guarantees we add a minute to the test run. I would utilize the timeout feature to wait for something that loads on the page that way we are waiting until an element loads instead of guessing how long it takes.

Reply:
ack